pkg/scheduler/objects/application_state.go

```diff
@@ -61,10 +63,11 @@ const (
 	Rejected
 	Completed
 	killed
+	Deleting
 )
 
 func (as applicationState) String() string {
-	return [...]string{"New", "Accepted", "Starting", "Running", "Waiting", "Rejected", "Completed", "killed"}[as]
+	return [...]string{"New", "Accepted", "Starting", "Running", "Waiting", "Rejected", "Completed", "killed", "Deleting"}[as]
```

Comment:
Suggest to rename `Deleting` to `Expired`.
And corresponding event rename to `expireApplication`

Reply:
Renamed it